CMakeLists.txt

```diff
@@ -24,13 +24,13 @@ SET(CMAKE_AUTOUIC ON)
 SET(CMAKE_AUTOMOC ON)
 SET(CMAKE_AUTORCC ON)
 
-SET(CMAKE_CXX_STANDARD 11)
+SET(CMAKE_CXX_STANDARD 17)
 SET(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 ENABLE_TESTING()
 FIND_PACKAGE(QGIS)
 
-SET(QT_MIN_VERSION 5.13.0)
+SET(QT_MIN_VERSION 5.14.0)
```

Comment:
Is this a requirement?

Reply:
I just used this PR to fix this. After the recent changes in TextEdit widget validators, it is a requirement.

Reply:
https://github.com/opengisch/QField/blob/cpp17_ftw/src/qml/editorwidgets/TextEdit.qml#L1